.travis.yml

```diff
@@ -11,7 +11,7 @@ addons:
     update: true
 before_install:
   - git config --global clangFormat.binary clang-format-9
-  - npm install -g markdownlint-cli
+  - gem install mdl
```

Comment:
Could we force an specific version for `mld`?

Reply:
For sure!